sources/engine/Stride.Engine/Rendering/Compositing/ForwardRenderer.cs

```diff
@@ -723,10 +723,6 @@ protected override void DrawCore(RenderContext context, RenderDrawContext drawCo
 
                     //draw mirror to backbuffer (if size is matching and full viewport)
                     if (VRSettings.CopyMirror)
-                    {
-                        CopyOrScaleTexture(drawContext, VRSettings.VRDevice.MirrorTexture, drawContext.CommandList.RenderTarget);
-                    }
-                    else if (hasPostEffects)
```

Comment:
I understand this is to fix the mirror issue you mentioned, why is this necessary exactly ?

Reply:
Also, is this fix influencing anything when running OpenVR?

Reply:
It's necessary because OpenXR doesn't have MirrorTexture implemented, and the OpenVR implementation is broken anyway. The whole logic there was flawed to begin with, because it meant that if you enabled PostFx, it would magically enable the Mirror, even if you had disabled it.

## Here's the results of all combinations for reference:
### VRSettings.VRDevice.MirrorTexture:
OpenXR + PostFx: Error on Startup (MirrorTexture doesn't exist)
OpenXR + no PostFx: Error on Startup (MirrorTexture doesn't exist)
OpenVR + PostFX: Bad crop - shows only upper left of each eye on Mirror
OpenVR + no PostFx: Bad crop - shows only upper left of each eye on Mirror

### vrFullSurface:
OpenXR + PostFx: Good
OpenXR + no PostFx: Good
OpenVR + PostFX: Good
OpenVR + no PostFx: Good